server.py

```diff
@@ -38,6 +38,10 @@ def parse_args():
         raise RuntimeError("This application requires Python 3.6+")
     parser = parse_args()
     args = parser.parse_args()
+    try:
+        args.debug=int(args.debug)
```

Comment:
are you sure it it's necessary ?

Reply:
Yes. it allows the user to use string mode of debug level (user friendly) or int value (aligned with the default in the argparse).